corehq/apps/importer/util.py

```diff
@@ -346,15 +347,37 @@ def get_spreadsheet(download_ref, column_headers=True):
         return None
     return ExcelFile(download_ref.get_filename(), column_headers)
 
+
+def is_location_group(owner_id, domain):
+    """
+    Return yes if the specified owner_id is one of the
+    faked location groups.
+    """
+    group_ids = [
```

Comment:
if not - it at least seems like you could pull out the id generation here and call it instead of referencing the object's ID.